mycroft/skills/intent_service.py

```diff
@@ -348,6 +353,17 @@ def handle_utterance(self, message):
             lang = _get_message_lang(message)
             setup_locale(lang)  # set default lang
 
+            if lang not in self.langs:
+                lang2 = lang.split('-')[0]
+                for supported_lang in self.langs:
+                    if supported_lang.startswith(lang2):
+                        lang = supported_lang
+                        break
+            if lang not in self.langs:
+                self.bus.emit(message.reply(
```

Comment:
this is dangerous to do and not localizable, speech should only be emitted by skills. 

This is also happening before fallback skills have a chance to answer, a best solution would be to make a new fallback skill

```python
class UnknownLangSkill(FallbackSkill):
    def initialize(self):
        self.register_fallback(self.handle_fallback, 99)

    def handle_fallback(self, message):
         if self.lang not in langs:
             self.speak_dialog("missing.language")
             return True
        return False

def create_skill():
    return UnknownLangSkill()
```